.github/workflows/ci.yml

```diff
@@ -31,6 +31,15 @@ jobs:
         run: |
           validate-mbz-html html errors.csv html
           CSV_LINES=$(wc -l < errors.csv )
+          if [[ $CSV_LINES != "1" ]]; then
+            echo "Validation errors found in html:"
+            cat errors.csv
+            exit 1
+          fi
+      - name: Validate html including tables
+        run: |
+          validate-mbz-html html errors.csv html --tables
+          CSV_LINES=$(wc -l < errors.csv )
```

Comment:
A few things:

1. This won't work / pass until we merge your PR in `raise-mbtools` (which is fine / expected)
2. We should print the number of errors found in the output. That way @mjoss17 / WW can track the count and make sure it's decreasing. I do realize that it's not going to be the count of broken tables, but instead the number of errors since we're creating an error for each violation, but I think that is okay 🤷‍♂️ .
3. I don't think we need the `if` statement here since no matter what we don't want to `exit 1`. We can just do something like (the `bash` may not be correct 😅 ):

```bash
validate-mbz-html html errors.csv html --tables
CSV_LINES=$(wc -l < errors.csv )
echo "$(($CSV_LINES-1)) table errors found in html"
```